haskell/1/index.md

```diff
@@ -119,40 +78,27 @@ bassbull project.
 
 ```
 dist
+dist-*
 cabal-dev
 *.o
 *.hi
 *.chi
 *.chs.h
-.virtualenv
+*.dyn_o
+*.dyn_hi
 .hpc
 .hsenv
 .cabal-sandbox/
 cabal.sandbox.config
-cabal.config
 *.prof
-*.hp
 *.aux
+*.hp
+*.eventlog
+.stack-work/
 ```
 
-You might be wondering why we're telling `git` to ignore something
```

Comment:
Seeing this diff makes me really happy at all the content that's no longer needed :)